RocketChat/Rocket.Chat
# fix: #28762 - fix missing command in /help

fix missing command for: "Mark all messages as read" - under /help
**closes** #28762 







## Proposed changes (including videos or screenshots)


When using `/help` command in any room the rocket.cat bot lists all keyboard shortcuts for the operations that can be done in rocket chat but for one command(marking all messages as read) shortcut is not being renderered

**before fix screenshot:**
<img width="653" alt="Screenshot 2023-04-03 at 23 49 21" src="https://user-images.githubusercontent.com/112387862/229480617-570d422c-6e94-4306-9733-4247f6dc9be1.png">

**post fix screenshot:**
<img width="653" alt="Screenshot 2023-04-03 at 23 49 21" src="https://user-images.githubusercontent.com/13575704/229593700-38c85b52-3bc2-4ae0-ba99-76bf7f0229db.png">



## Issue(s)

closes #28762 

## Steps to test or reproduce

* open any channel
* type "/help" and press enter(return) key
* bot responds with a list of commands with their description
* notice here for 2nd command - "Mark all messages as read" - the shortcut command is missing

all commands must have the shortcut command as well

## Further comments

this PR adds the missing placeholder for mark all messages as read command for English(en) language
